File: Perl/fizzbuzz.pl
#!/usr/bin/perl
use strict;
use warnings;

for (my $count = 1; $count <= 100; $count++) {
	if ($count%15 == 0) 
	{
		print "FIZZBUZZ\n";
	} 
	elsif ($count%3 == 0) 
	{
		print "FIZZ\n";
	} 
	elsif ($count%5 == 0) 
	{
		print "BUZZ\n";
	} 
	else 
	{
		print "$count\n";
	}

}

File: Ruby/fizzbuzz.rb
#!/usr/bin/ruby

#FIZZBUZZ Ruby
for i in (1..100) do
	if i%15 == 0 
		puts "FIZZBUZZ"
	elsif i % 3 == 0 
		puts "FIZZ"
	elsif i % 5 == 0 
		puts "BUZZ"
	else 
		puts i
	end
end

